packages/db/src/rules.ts

```diff
@@ -0,0 +1,45 @@
+import Knex from 'knex';
+import { RuleRecord } from '@cumulus/types/api/rules';
+import { PostgresRule, PostgresCollectionRecord, PostgresProviderRecord } from './types';
+import { getRecordCumulusId } from './database';
+import { tableNames } from './tables';
+
+/**
+ * Generate a Postgres rule record from a DynamoDB record.
+ * @param {Object} record - A rule
+ * @param {Object} dbClient - Knex client for reading from RDS database
+ * @returns {Object} A rule record
+ */
+export const translateApiRuleToPostgresRule = async (
+  record: RuleRecord,
+  dbClient: Knex
+): Promise<PostgresRule> => {
+  const ruleRecord: PostgresRule = {
+    name: record.name,
+    workflow: record.workflow,
+    provider_cumulus_id: record.provider ? await getRecordCumulusId<PostgresProviderRecord>(
+      { name: record.provider },
+      tableNames.providers,
+      dbClient
+    ) : undefined,
+    collection_cumulus_id: record.collection ? await getRecordCumulusId<PostgresCollectionRecord>(
+      { name: record.collection.name, version: record.collection.version },
+      tableNames.collections,
+      dbClient
+    ) : undefined,
+    meta: record.meta,
+    payload: record.payload as any,
```

Comment:
Is `as any` necessary?

Reply:
Yes I believe so because the type is `unknown` which isn't compatible with `object` 
https://github.com/nasa/cumulus/blob/master/packages/types/api/rules.d.ts#L26

Reply:
[`717d0e5` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/717d0e50dc816dd503b437d92a583a563a6c0766)